app/build.gradle.kts

```diff
@@ -188,39 +188,13 @@ dependencies {
     androidTestImplementation(libs.androidx.test.rules)
     androidTestImplementation(libs.androidx.test.junit)
 
-    implementation(libs.androidx.other.appCompat)
-    implementation(libs.androidx.other.media)
-    implementation(libs.androidx.other.constraintLayout)
-    implementation(libs.androidx.other.dynamicAnimation)
-    implementation(libs.androidx.other.preference)
-    implementation(libs.androidx.other.core)
-    implementation(libs.androidx.other.viewPager2)
-    implementation(libs.androidx.other.swipeRefreshLayout)
-    implementation(libs.androidx.other.recyclerView)
-    implementation(libs.androidx.other.fragment)
-    implementation(libs.androidx.other.browser)
-    implementation(libs.androidx.lifecycle.common)
-    implementation(libs.androidx.lifecycle.liveData)
-    implementation(libs.androidx.lifecycle.runtime)
-    implementation(libs.androidx.lifecycle.viewModel)
-    implementation(libs.androidx.other.workManager)
-    implementation(libs.androidx.datastore.core)
-    implementation(libs.androidx.datastore.preferences)
-    implementation(libs.androidx.other.startup)
-    androidTestImplementation(libs.androidx.espresso.core)
-    androidTestImplementation(libs.androidx.espresso.intents)
-    androidTestImplementation(libs.androidx.espresso.contrib)
-    testImplementation(libs.androidx.test.junit)
-    testImplementation(libs.jsonTest)
-    androidTestImplementation(libs.androidx.test.runner)
-    androidTestImplementation(libs.androidx.test.rules)
-    androidTestImplementation(libs.androidx.test.junit)
-
```

Comment:
I happened to be in the build.gradle.kts and I saw that we have all these written twice, must have been when I merged some stuff and messed this up 😅